packages/backend/src/apps/formsg/auth/decrypt-form-response.ts

```diff
@@ -54,13 +50,15 @@ export async function decryptFormResponse(
     return false
   }
 
+  const formSgSdk = getSdk(parseFormEnv($))
```

Comment:
```suggestion
  const formsgSdk = getSdk(parseFormEnv($))
```
super nit seems like formsg should just all be small letters? up to you I'm just putting it out here